### ISSUE TITLE
Emoji grid is empty

### ISSUE THREAD

**Issue by saeed-rz** [rockerhieu/emojicon#121]
i add fragment to layout but emoji grid is empty!!
emoji tabs show and change when slide to side but nothing show in grid!

**Comment by saeed-rz** [comment]
sorry, i think this is my mistake!
when i use 

> getSupportFragmentManager()
>                 .beginTransaction()
>                 .replace(R.id.emojicons, EmojiconsFragment.newInstance(useSystemDefault))
>                 .commit();

emoji item disappear.! 
it's ok now :+1:

**Comment by saeed-rz** [comment]
#122

**Comment by rockerhieu** [comment]
@saeed-rz can you send a minimum example project here?

**Comment by saeed-rz** [comment]
@rockerhieu yes i can, but currently i am very busy, i try send sample project very soon

**Comment by saeed-rz** [comment]
sorry for late

**Comment by saeed-rz** [comment]
where should i upload sample project?

**Comment by rockerhieu** [comment]
Here!

**Comment by saeed-rz** [comment]
i cant upload zip file here, so uploaded on this server
[source](http://164.138.23.237/src.zip)
the zip file contain "src" folder of project
the fragment show at the bottom of view, and with some customization can show emoji on keyboard position
hope its helpful

**Comment by saeed-rz** [comment]
is sample project helpful?
